src/main/java/org/omnifaces/util/Components.java

```diff
@@ -248,6 +248,18 @@ public static <T> T getAttribute(UIComponent component, String name) {
         return (T) component.getAttributes().get(name);
     }
 
+    /**
+     * Set the attribute value on the specified component for the specified name.
+     * @param <T> The expected value type.
+     * @param component The component on which to set the attribute.
+     * @param name The name of the attribute.
+     * @param value The value of the attribute.
+     * @since 4.5
+     */
+    public static <T> void setAttribute( UIComponent component , String name , T value ) {
```

Comment:
Can you please follow the same code format as seen in existing source code? The whitespace is off here.

Reply:
Using `<T>` is unnecessary here because it's unrestricted in parameter value and unused in return value. It can as good be `Object`.

Reply:
ok, it's only a personal preference... recently I've adopted the "generics style" for uniformity with methods that return the `<T>` `T` :)